pkg/ccl/streamingccl/streamproducer/replication_stream_test.go

```diff
@@ -229,6 +230,42 @@ func testStreamReplicationStatus(
 	checkStreamStatus(t, hlc.MaxTimestamp, expectedStreamStatus)
 }
 
+func TestSpanConfigReplicationStreamSetup(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	serverArgs := base.TestServerArgs{
+		// This test fails when run from within a test tenant. This is likely
+		// due to the lack of support for tenant streaming, but more
+		// investigation is required. Tracked with #76378.
+		DefaultTestTenant: base.TODOTestTenantDisabled,
+		Knobs: base.TestingKnobs{
+			JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals(),
+		},
+	}
+
+	h, cleanup := replicationtestutils.NewReplicationHelper(t, serverArgs)
+	defer cleanup()
+	testTenantName := roachpb.TenantName("test-tenant")
+	_, cleanupTenant := h.CreateTenant(t, serverutils.TestTenantID(), testTenantName)
+	defer cleanupTenant()
```

Comment:
We don't really need to even start the tenant sql server for this this. I think we could just `CREATE TENANT $1;`